website/www/site/data/en/calendar_events.yaml

```diff
@@ -0,0 +1,27 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- title: This is the title
+  place: Data Con LA
+  time: 2020/02/21
+  icon: icons/calendar-icon.svg 
+  url: #
+- title: This is the title
+  place: Data Con LA
+  time: 2020/02/21
+  icon: icons/calendar-icon.svg 
+  url: #
+- title: This is the title
+  place: Data Con LA
+  time: 2020/02/21
```

Comment:
Is it possible for the dates to be represented as some kind of timestamp literal that can be internationalized? For example in the US we usually write dates with the year last - 02/21/2020. Or we may want to have the option to spell out the date in the future.

Reply:
Noted. Thanks for your feedback. Let me tell my PM to add them into Jira.